oeps/architectural-decisions/oep-0052-arch-event-bus-architecture.rst

```diff
@@ -77,6 +77,10 @@ Decision
   * See edx.org `ADR on Kafka Managed Hosting`_ if considering Kafka hosting options.
   * See edx.org `ADR on Managing Kafka Consumers`_ if exploring how to deploy consumers for Kafka.
 
+* A secondary implementation is being implemented using Redis streams.
```

Comment:
Should we change this and the above sentence as follows?
```suggestion
* A secondary implementation has been implemented using Redis streams.
```

Reply:
@robrap Yes. Should we also update the same for Kafka [here](https://github.com/openedx/open-edx-proposals/blob/33c1cabcf3d7e4f95c57baa9365f17bbab465e16/oeps/architectural-decisions/oep-0052-arch-event-bus-architecture.rst?plain=1#LL74C3-L74C72). Actually, most of the functionality written as being/will be implemented in the doc has been completed.

Reply:
I think so. It doesn't change the decision, it just makes it easier to understand status.

Reply:
Updated.